src/plugins/dashboard/public/application/legacy_app.js

```diff
@@ -94,7 +94,7 @@ export function initDashboardApp(app, deps) {
       .when(DashboardConstants.LANDING_PAGE_PATH, {
         ...defaults,
         template: dashboardListingTemplate,
-        controller($scope, kbnUrlStateStorage, history) {
+        controller: function($scope, kbnUrlStateStorage, history) {
```

Comment:
this `function` changes something? or no need?

Reply:
Ohhh, that's a very interesting point!
So, it was working fine in legacy platform, but stop working in the new platform.
The failure was something like: `Angular can't bootstrap...`, the reason was the controller is invoked as `new controller()` inside the angular and this particular function was not converted to a function declaration and acts like an arrow function.
So seems we have different babel configs for legacy and new platform (in legacy object methods are converted to function declarations, but in new platform they are not).
Maybe @joshdover could clarify this? 🙂